flixel/addons/editors/tiled/TiledImageSet.hx

```diff
@@ -0,0 +1,51 @@
+package flixel.addons.editors.tiled;
+
+import haxe.xml.Fast;
+
+/**
+ * Copyright (c) 2013 by Samuel Batista
+ * (original by Matt Tuttle based on Thomas Jahn's. Haxe port by Adrien Fischer)
+ * This content is released under the MIT License.
+ */
+class TiledImageSet implements Dynamic<String>
```

Comment:
This is a 1:1 copy of `TiledPropertySet` except for the implementation of `extend()` - a common base class might not be a bad idea.